### DISCUSSION TITLE
Icons and Colors missing.

### DISCUSSION THREAD

**Discussion by Bhagathxoxo** [Jisco/VisualFamilyTree#25] [Ideas]
- It is confusing to understand quickly whether they are Husband and wife or Siblings. Visually lines are hard to process quickly. Adding an icon or colors will be useful. 

- If I want to add sibling, I have to select parent and add kid, Adding an extra option for sibling will be mush faster.

Also is there any way to view or export the full chart instead of viewing only the main character tree?

**Comment by Jisco** [comment]
> It is confusing to understand quickly whether they are Husband and wife or Siblings. Visually lines are hard to process quickly. Adding an icon or colors will be useful.

Actually partner have an golden border. Everyone with an black border is an sibling.
![image](https://github.com/Jisco/VisualFamilyTree/assets/770001/11fd5c41-f57f-42f0-b60a-0cc1c7a4f2f4)

> If I want to add sibling, I have to select parent and add kid, Adding an extra option for sibling will be mush faster.

Yes, i try this to remember for later development.

> Also is there any way to view or export the full chart instead of viewing only the main character tree?

No, sorry. This is actually not possible. The problem is that in this case a node can have multiple parents so it wont be a tree anymore and i dont have found an good way to calculate this graphs yet. But its definitly on my list for an next version.

**Reply by Bhagathxoxo** [reply]
You are doing an awesome work. Thank you for your response.
